vars/rpmTestVersion.groovy

```diff
@@ -11,5 +11,8 @@
  */
 
 boolean call() {
+    if (params.CI_RPM_TEST_VERSION) {
+        return params.CI_RPM_TEST_VERSION
+    }
```

Comment:
This doesn't seem to agree with https://github.com/daos-stack/pipeline-lib/blob/27ccb295de9ce63fd80a8752621220bde6974678/vars/skipStage.groovy#L43.

But I'm also not sure that pragmas should have the highest priority. It has been expressed by others that they were surprised that the existing parameters we have did not override commit pragmas.

Reply:
Yeah, that is inconsistent.  Would prefer to make it consistent in a later update if possible.

Reply:
How much later?  I just don't want anyone stumbling on to the inconsistency and complaining.  Let's at least have a ticket we can point them at.

But to be clear, I think parameters (where specified, and commit pragams should be honoured where a parameter is not specified) should override commit pragmas as the PoR.

Reply:
Updated comment for skip_ftest.  Effectively the checks need to be done for pragmas to enabled skipped functional tests before checks are done for parameters to enable skipped functional checks.